app/controllers/entries_controller.rb

```diff
@@ -32,7 +33,8 @@ def entry_params
       :daily_deposit_count, :interest_level, :no_of_locations,
       :alajo_name, :alajo_phone, :alajo_prime_location, :alajo_no_of_locations,
       :alajo_union, :alajo_population, :alajo_avg_age, :alajo_avg_customer,
-      :alajo_avg_daily_contr, :alajo_avg_acc_bal
+      :alajo_avg_daily_contr, :alajo_avg_acc_bal,
+      locations_attributes: [:location_platform, :location_est, :location_geotag]
```

Comment:
accept the location attributes.
Enabled because of the `accepts_nested_attributes_for :locations` in entry.rb

Reply:
Like i said, if you don't want to use Rails form/helper, crafting the nested attributes is as simple as:
```ruby
{ 
  'entry' => {
    'locations_attributes' => {
      '0' => {
        'location_platform' => 'monniepoint',
        'location_est' => '1990',
        'location_geotag' => '6789'
        }, 
      '1602651569105' => {
        'location_platform' => 'paga',
        'location_est' => '1i373',
        'location_geotag' => '27842'
      }
    }
  }
}
```
See the index `0` as the pre-built location, while the `1602651569105` is the timestamped that was used to replace the value in the template